tests/py/test_markdown.py

```diff
@@ -6,50 +7,99 @@
 
 class TestMarkdown(Harness):
 
-    # render
+    # r - render
```

Comment:
... aaaaaaaaand addressed in 7e6778c16d42e720c33b330539afe645b5f8e494. :-)

Easy enough to do `py.test test_markdown.py::TestRender`.